earthfile2llb/interpreter.go

```diff
@@ -1127,6 +1128,18 @@ func (i *Interpreter) handleSaveImage(ctx context.Context, cmd spec.Command) err
 		fmt.Fprintf(os.Stderr, "Deprecation: using SAVE IMAGE with no arguments is no longer necessary and can be safely removed\n")
 		return nil
 	}
+
+	labels := make(map[string]string)
+	labels["dev.earthly.version"] = version.Version
+	labels["dev.earthly.git-sha"] = version.GitSha
+	labels["dev.earthly.built-by"] = version.BuiltBy
+	labels["dev.earthly.default-installation-name"] = version.DefaultInstallationName
```

Comment:
What's the use-case for storing the default install name (and buildkitd image)? These values can be overridden at run time (or even could be using a remote buildkit instance, e.g. satellite, which could be a different version).

Reply:
No use case, just added them there for completion. Wasn't aware they are mutable